<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebMapDialog</class>
 <widget class="QDialog" name="WebMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Дорожно-транспортные происшествия на карте</string>
  </property>
  <widget class="QLabel" name="label_dateto">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>5</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>по</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditfrom">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>5</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_resetfilter">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>5</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Сброс</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_datefrom">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>5</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Дата ДТП с</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_applyfilter">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>5</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_showfilter">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>5</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Фильтр по дате</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_to">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>5</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
